src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeSlider.qml

```diff
@@ -74,9 +74,7 @@ StyledSlider {
 
         onReleased: {
             prv.dragActive = false
-            if (!containsMouse) {
-                tooltip.hide(true)
-            }
+            tooltip.hide(true)
```

Comment:
I remember we explicitly wanted the tooltip to stay on as long as the mouse hovered over the slider.